### DISCUSSION TITLE
Group field as required in user creation (admin console)

### DISCUSSION THREAD

**Discussion by pepesoriagarcia99** [keycloak/keycloak#16832] [Archive]
How can I make the group field mandatory in the user creation form?

**Comment by ssilvert** [comment]
@pepesoriagarcia99 You should be able to achieve this if you follow a similar setup to the one described in https://github.com/keycloak/keycloak-ui/issues/3965

The administrator will get an error if he doesn't assign the user to one of his authorized groups.  It's not the cleanest user experience, but it should work.
